sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -361,6 +407,31 @@ export class AssessmentControllerService {
     return finalAward;
   }
 
+  /**
+   * Populate final awards values from e-Cert effective values.
+   * @param disbursementValues disbursement values with the effective amounts used to generate the e-Cert.
+   * @param identifier identifier which is used to create dynamic data by appending award code to it.
+   * @returns dynamic award data of disbursement receipts of a given disbursement.
+   */
+  private populateDisbursementECertAwardValues(
+    disbursementValues: DisbursementValue[],
```

Comment:
Minor formatting error.

![image](https://github.com/bcgov/SIMS/assets/54600590/40d35e78-09f2-425d-b73d-a5e31cb46415)

Reply:
How, I have no idea. Just did the same and it formatted as expected.